### PR TITLE
Render for Blod, HorizontalRule and FenchedCodeBlock

### PR DESCRIPTION
**Pull request by jinglongwu**
like this?

### CODE REVIEW

File: test_bold_element.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from bold_element import BoldElement
```
Comment (andysim3d) [inline test_bold_element.py]: 文件名后缀是_test.py

File: bold_element.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from element import Element
+import re
+
+class BoldElement(Element):
+
+    def _render(self):
+        pattern = re.compile(r'[*]{2}(.*?)[*]{2}')
```
Comment (andysim3d) [inline bold_element.py]: 你是打算现在就写parser?

File: bold_element.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+        rendered = pattern.sub(r'<strong>\1</strong>', self.content())
```
Comment (andysim3d) [inline bold_element.py]: [Opt]: better have a space between label and content.

File: bold_element.py (hunk 2 above)
Comment (jinglongwu) [reply bold_element.py]: 哈哈哈忘了只用写render。 那就是return `"<strong>{}<strong>".format(self.content())`?

File: bold_element.py (hunk 3 above)
Comment (jinglongwu) [reply bold_element.py]: You mean `<strong> \1 </strong>` rather than `<strong>\1</strong>`, right?

File: bold_element.py (hunk 2 above)
Comment (andysim3d) [reply bold_element.py]: 嗯， 现在先不考虑render。而且block应该不需要render children（没有children）

File: bold_element.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+from element import Element
+import re
+
+class BoldElement(Element):
```
Comment (andysim3d) [inline bold_element.py]: 你可以先pull一下我的PR，我定义了俩子类，用来代表Block & Paragraph。这个可以写成BoldBlock()

File: bold_element.py (hunk 3 above)
Comment (andysim3d) [reply bold_element.py]: Yes, but its totally optional.

Comment (VCneverdie) [reply bold_element.py]: I think the whitespace matters. `my<strong>test</strong>` and `my<strong> test </strong>` are two different things.

Comment (jinglongwu) [reply bold_element.py]: > I think the whitespace matters. `my<strong>test</strong>` and `my<strong> test </strong>` are two different things.

that's right. there would be extra space in the html text

**Comment by jinglongwu** [comment]
@andysim3d @VCneverdie Please review again.

**Review by andysim3d — APPROVED** [review]
too many blank lines after code.... otherwise LGTM

**Pull request merged**